pontoon/machinery/openai_service.py

```diff
@@ -20,29 +21,32 @@ def get_translation(
                 f"The target language '{target_language_name}' is not supported."
             )
 
-        informal = (
-            f"You will be provided with text in English, along with its machine-generated translation in {target_language}. "
+        informal = textwrap.dedent(
+            f"""
+            You will be provided with text in English, along with its machine-generated translation in {target_language}. "
```

Comment:
The trailing quote here and the quotes on the following lines are now a part of the string value.